libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -788,6 +788,7 @@ ap_message GCS_MAVLINK::mavlink_id_to_ap_message_id(const uint32_t mavlink_id) c
         { MAVLINK_MSG_ID_EXTENDED_SYS_STATE,    MSG_EXTENDED_SYS_STATE},
         { MAVLINK_MSG_ID_AUTOPILOT_VERSION,     MSG_AUTOPILOT_VERSION},
         { MAVLINK_MSG_ID_EFI_STATUS,            MSG_EFI_STATUS},
+        { MAVLINK_MSG_ID_HYBRID_READ,           MSG_GENERATOR_READ},
```

Comment:
Hmm.. and now we use the word "HYBRID", I think it should be "GENERATOR" if we are only sending information about the generator.  If we were sending information about both gas and electric drive trains then maybe Hybrid would be OK.. but to me it seems more like we're somewhat arbitrarily using different words for the same thing.

Reply:
We're going to look at reworking messaging to use the new mavlink generator messages instead.